### DISCUSSION TITLE
Next13 cannot uplaod image

### DISCUSSION THREAD

**Discussion by jaywei1234** [pocketbase/pocketbase#1852] [Q&A]
Here are my codes.
![image](https://user-images.githubusercontent.com/56056463/218720540-3d5049f3-c30a-4f0a-8a73-649e937ba55f.png)
![image](https://user-images.githubusercontent.com/56056463/218720260-fee46aa7-f8ed-404d-a790-e1ffb4ac3bc7.png)
![image](https://user-images.githubusercontent.com/56056463/218720326-d81fd3d5-aba4-498f-a0b0-c7a054fc8f39.png)
![image](https://user-images.githubusercontent.com/56056463/218720489-07021902-2b5e-4d1d-b33b-2bbb465e5543.png)

After I submit the form, only the image file didn't uploaded to PocketBase, I don't know how to fix it on Next13. Thank you!

**Comment by ganigeorgiev** [comment]
I'm not able to reproduce the issue locally with plain JS.

Are you sure that `thumbnail` is a `File` or `Blob` instance and it is not empty?

Could you inspect the Network request payload in your browser's "Inspect Tool > Network > {find the create request} > Payload" and check if thumbnail is marked as "binary" (_you can also click on "view source" to inspecting the raw `multipart/form-data` request body_)?
